README.md

```diff
@@ -316,6 +316,31 @@ setVerbose(True)
 setVerbose(False)
 ```
 
+
+### Output Files
+Batch will automatically handle your output files when the user assigns a file pattern and container url. 
+
+```R
+# Pushing output files
+storageAccount <- "storageAccountName"
+outputFolder <- "outputs"
+
+createContainer(outputFolder)
+writeToken <- rAzureBatch::createSasToken("w", "c", outputFolder)
+containerUrl <- rAzureBatch::createBlobUrl(storageAccount = storageAccount,
+                                           containerName = outputFolder,
+                                           sasToken = writeToken)
+
+output <- createOutputFile("test-*.txt", containerUrl)

```

Comment:
These samples/docs are **_really_** important for our users since it's the only way they'll know to use this. Can we add a few more examples on how you might construct output files? Perhaps some more complicated use cases would be good to show the flexibility of it.